archive/j/java/EvenOdd.java

```diff
@@ -0,0 +1,30 @@
+public class EvenOdd
+{
+    public static void verifyNumber(String n) throws Exception
+    {
+        char[] nArray = n.toCharArray();
+        for (char c : nArray)
+        {
+            if (!Character.isDigit(c))
+                throw new Exception();
```

Comment:
Can you try throwing something more specific like https://docs.oracle.com/javase/8/docs/api/java/lang/NumberFormatException.html?